docs/tempo/website/getting-started/_index.md

```diff
@@ -39,7 +39,7 @@ information from a client application with minimal manual instrumentation of the
 * [OpenTemeletry .NET Autoinstrumentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation)
 * [OpenTemeletry Python Autoinstrumentation](https://github.com/open-telemetry/opentelemetry-python-contrib)
 
-> Note: Check out our [instrumentation examples]({{< relref "../guides/instrumentation" >}}) to learn how to instrument your
+> Note: Check out our [instrumentation examples]({{< relref "./instrumentation" >}}) to learn how to instrument your
 > favourite language for distributed tracing.
 
 ## 2. Pipeline (Grafana Agent)
```

Comment:
(Github wont let me add a review comment at another position in this file)

at this location - https://github.com/grafana/tempo/pull/734/files#diff-3565f24807ee62ad548ce0767d5b214da9ac3e51fa0501129a53034bdfceb314R73

`../deployment` => `../operations`